### ISSUE TITLE
Agent with many dofs: Discriminator accuracy suddenly drops to zero

### ISSUE THREAD

**Issue by braunjon** [nv-tlabs/ASE#52]
Hi,

I am trying to train a low-level policy with an agent that has roughly 100 dofs.
But after a few hundred iterations the policy only predicts the zero action (almost no movement) and the discriminator accuracy suddenly  drops on the agent data from around 0.99 to zero while the accuracy on the demo data stays at 1. Also sometimes the ratio in the actor loss is inf.

While the zero pose is part of the reference motion, it looks otherwise fine to me. Also if I train the same agent reduced to around 50 dofs and the same reference motion it works.

Could you maybe share some insights if possible. Maybe on which hyper parameters to tune when there are a lot of dofs?

Thank you!

**Comment by xbpeng** [comment]
A sudden large drop in the accuracy like that is suspicious, as well as the inf in the action ratio. This is probably a sign that the hyperparameters need to be tuned. You might want to take a look at the behavior before and after the drop in accuracy and see what the difference is. If the number of dofs are different, you likely need to tune the learning rate and gradient penalty.
